webpack/webpack.dev.babel.js

```diff
@@ -0,0 +1,107 @@
+import ExtractTextPlugin from 'extract-text-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import StringReplacePlugin from 'string-replace-webpack-plugin';
+import WebpackNotifierPlugin from 'webpack-notifier';
+
+import packageInfo from '../package';
+import webpackConfig from './webpack.config.babel';
+
+let PORT = process.env.PORT || 8080;
+let proxy;
+
+try {
+  // If this fails - you should create a proxy.dev.js by copying proxy.template.js
+  // and customizing the proxy as needed.
+  proxy = require('./proxy.dev.js');
+} catch (err) {
+  proxy = require('./proxy.template.js');
+}
+
+let REPLACEMENT_VARS = {
+  VERSION: packageInfo.version,
+  ENV: process.env.NODE_ENV,
+  ANALYTICS_KEY: packageInfo.analytics.development
+};
+
+module.exports = Object.assign({}, webpackConfig, {
+  entry: [
+    'webpack-dev-server/client?http://localhost:' + PORT,
+    'webpack/hot/only-dev-server',
+    './src/js/index.js'
+  ],
+  devtool: '#eval-source-map',
+  output: {
+    path: './build',
+    filename: 'index.js'
```

Comment:
What do you think about getting rid of the `build.config.js` file? Those variables aren't really needed because we only use them here. This webpack config is for this repo and doesn't need to be generalized.